lib/net.c

```diff
@@ -25,11 +25,43 @@
 #include "gatekeeper_net.h"
 #include "gatekeeper_config.h"
 
+/*
+ * The maximum number of "rte_eth_rss_reta_entry64" structures can be used to
+ * configure the Redirection Table of the Receive Side Scaling (RSS) feature.
+ * Notice, each "rte_eth_rss_reta_entry64" structure can configure 64 entries 
+ * of the table. To configure more than 64 entries supported by hardware,
+ * an array of this structure is needed.
+ */
+#define GATEKEEPER_RETA_MAX_SIZE (ETH_RSS_RETA_SIZE_512 / RTE_RETA_GROUP_SIZE)
+
 static struct net_config config;
 
+/*
+ * XXX The RSS secret key with a length of 40 bytes, and it's used by the RSS hash.
+ * Change the key if we need some special features, e.g., symmetric RSS.
+ */
```

Comment:
This is a good suggestion! Here, I used ``XXX`` to imply for possible new features or improvements, and important suggestions. That is, we may need to change the key if we need some special features, e.g., symmetric RSS. Maybe we can keep both two suggestions?